### DISCUSSION TITLE
WLBridge4TRX - Connect your radio to Wavelog without a PC

### DISCUSSION THREAD

**Discussion by zone11** [wavelog/wavelog#546] [Show and tell]
I love portable operations and wanted to have my radios connected to Wavelog without the need of a PC. The result is a few lines of code running on an ESP32 to read the current frequency and mode from my radios using CAT. The ESP32 connects itself to my Wavelog installation over WIFI (using my iPad's hotspot) and uses the API for Hardware Interfaces.

If somebody is interested in having something similar, feel free to have a look at https://github.com/zone11/WLBridge4TRX
Tested and operational with YAESU und ELECRAFT radios, HTTPS is supported 😉

73 de HB9HJQ

![ESP32Thing-KX2](https://github.com/wavelog/wavelog/assets/1774084/08305649-eaaf-4e0e-819f-f571360fd36a)

**Comment by HB9HIL** [comment]
Awesome! 😎👍
